src/hotspot/share/gc/parallel/parallelScavengeHeap.cpp

```diff
@@ -66,7 +66,7 @@ jint ParallelScavengeHeap::initialize() {
   os::trace_page_sizes("Heap",
                        MinHeapSize,
                        reserved_heap_size,
-                       GenAlignment,
+                       os::vm_page_size(),
```

Comment:
I briefly looked at this in the past and I don't think using `os::vm_page_size()` straight off will not work for the case when large pages are used. So we need a way to figure out if the heap got reserved using large pages. Once again you can take a look at how G1 does it, it's in `actual_reserved_page_size()` in g1CollectedHeap.cpp. I'm not sure you can use that straight off, but something similar should work.

Reply:
Thanks for comments! Have added a ParallelInitLogger now.